packages/integrations/vercel/src/edge/adapter.ts

```diff
@@ -39,6 +39,13 @@ export default function vercelEdge(): AstroIntegration {
 				_config = config;
 				serverEntry = config.build.serverEntry;
 				functionFolder = config.build.server;
+
+				if (config.output === 'static') {
+					throw new Error(`
+		[@astrojs/vercel] \`output: "server"\` is required to use the edge adapter.
+	
+	`);
+				}
```

Comment:
Should we have a warning instead similar to the other edge integrations?

https://github.com/withastro/astro/blob/f49d80d637e7ba6598139877b1081d7983d41e6c/packages/integrations/netlify/src/integration-functions.ts#L45-L50

Reply:
This adapter handles static deploys differently, that's why it throws an error. When you add it, you specify "I want to deploy to server less/edge"